controllers/flux/job.go

```diff
@@ -93,3 +94,13 @@ func (r *MiniClusterReconciler) getMiniClusterContainers(cluster *api.MiniCluste
 	}
 	return containers
 }
+
+// Function to return list of objects references for
+// imagePullSecrets. Current Spec only allows for a
+// single secret to be used. 
+func getImagePullSecret(cluster *api.MiniCluster) []corev1.LocalObjectReference {
```

Comment:
This will work with the original example if it's not defined / empty? I'm almost wanting to be more explicit - e.g., check to see if the secret isn't equal to the empty string, and only append to an empty list of references if it is defined.

And can we add a commented out line for the usage into our example yaml config (akin to the others)?

Reply:
Yes, from my testing, it looks like it will work if not defined, it would be nice to get verification on that though!

Reply:
Just added a commented out pull secret and explanation to the example! 😄

Reply:
Definitely! I'll test and verify right after dinner, and if all looks good this should be good to merge!

Reply:
Ok didn't want to leave you hanging - just tested (with our original job) and it works great! Let's :ship: it!